config.go

```diff
@@ -329,17 +329,19 @@ func loadAndValidateConfig(interceptor signal.Interceptor) (*Config, error) {
 		os.Exit(0)
 	}
 
+	// Before we validate the config, we first hook up our own loggers.
+	// This must be done before the config is validated if LND is running
+	// in integrated mode so that the log levels for various non-LND related
+	// subsystems can be set via the `lnd.debuglevel` flag.
+	SetupLoggers(preCfg.Lnd.LogWriter, interceptor)
```

Comment:
Nice :tada:

Reply:
yeah Im glad it ended up just being a simple fix :)